android/src/main/java/com/cotecna/capacitor/mediastore/Mediastore.java

```diff
@@ -0,0 +1,119 @@
+package com.cotecna.capacitor.mediastore;
+
+import android.content.ContentResolver;
+import android.content.ContentValues;
+import android.content.Context;
+import android.net.Uri;
+import android.provider.MediaStore;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
+public class Mediastore {
+
+    public String savePicture(Context context, String album, String filename, String path) throws Exception {
```

Comment:
We should implement the same on the iOS part to keep the same methods, models, params, etc.. 
This is an hybrid library and if something changes for android, it should change on iOS too, If not when importing this version of the library and executing the project on iOS device it will not work.